build_daemon/lib/client.dart

```diff
@@ -96,14 +99,17 @@ class BuildDaemonClient {
   void registerBuildTarget(BuildTarget target) => _channel.sink.add(jsonEncode(
       _serializers.serialize(BuildTargetRequest((b) => b..target = target))));
 
-  /// Builds all registered targets.
+  /// Builds all registered targets, including those not from this client.
```

Comment:
Probably worth noting the behavior if there is an ongoing build already (I assume it waits)